include/rocksdb/file_checksum.h

```diff
@@ -22,9 +22,13 @@ namespace ROCKSDB_NAMESPACE {
 constexpr char kUnknownFileChecksum[] = "";
 // The unknown sst file checksum function name.
 constexpr char kUnknownFileChecksumFuncName[] = "Unknown";
+// The default DB file checksum function name.
```

Comment:
What do you mean default? This seems confusing to me.

Reply:
Yeah, I feel the same. It is more precisely the "default" checksum function provided by the DB if one would like to use file checksum. Didn't figure out a better name and felt this name is "symmetric" to the one used in backup. I think I should update the inline code comment.

Reply:
I still don't believe "provided by the DB", but I do believe it's provided by the RocksDB API. You have to call GetFileChecksumGenCrc32cFactory() and pass that to DB options for it to be used, right? Can we say "standard" instead of "default"?